infrastructure/variables.tf

```diff
@@ -37,13 +43,27 @@ variable "jenkins_AAD_objectId" {
 variable "external_host_name" {
   default = ""
 }
+
 variable "subscription" {}
 
 variable "appinsights_instrumentation_key" {
   description = "Instrumentation key of the App Insights instance this webapp should use. Module will create own App Insights resource if this is not provided."
   default = ""
 }
 
+variable "asp_rg" {
+  default = ""
+}
+
+variable "asp_name" {
+  default = ""
+}
+
+variable "product_group_object_id" {
+  default = "c9ab670f-8f92-4170-ba08-796ccab27751"
```

Comment:
Why having this the default value and what it points to?

Reply:
this is the project group that was created for us ticket RDO-2664